templates/pool.conf.j2

```diff
@@ -0,0 +1,263 @@
+# {{ ansible_managed }}
+;;;;;;;;;;;;;;;;;;;;;;
+; Pool Configuration ;
+;;;;;;;;;;;;;;;;;;;;;;
+
+; The address on which to accept FastCGI requests.
+; Valid syntaxes are: 'ip.add.re.ss:port', 'port', '/path/to/unix/socket'.
+; This option is mandatory for each pool.
+listen = {{ listen | mandatory }}
```

Comment:
I think you mean `pool.listen`. Otherwise the dict listen dict will be inserted here